docusaurus/observability-best-practices/README.md

```diff
@@ -0,0 +1,57 @@
+# Observability Best Practices
+
+## Welcome
+
+This is the source for the [AWS Observability Best Practices site](https://aws-observability.github.io/observability-best-practices/). Everyone is welcome to contribute here, not just AWS employees!
+
+## How to run/develop this site
```

Comment:
Maybe we should add a pre-requisite section with node, yarn or npm?

Reply:
changes done